src/app/shared/components/managecitations/managecitations.component.ts

```diff
@@ -125,10 +128,12 @@ export class ManageCitationsModal implements OnInit, OnDestroy {
     }
 
     public getCitations() {
-        this._settingsService.getEntities(this.configSettings.citationURL)
+        this._settingsService.getEntitiesCitations(this.configSettings.citationURL)
             .subscribe(res => {
                 this.citations = res;
-                this.filteredData = this.citations;
+                this.filteredData = this.citations.filter(function (filter) {
+                    return filter != null;
+                });
```

Comment:
I wonder if it's better here to, instead of creating a new item in the settings service just for this one call, just make an http call here instead.